IA-2150 fix timeout on derrived instance generation

olethanh opened the pull request:
by increasing the general timeout
The corresponding modification has been done on the load balancer for prod This is a short term fix, the generation should be moved in a background task

olethanh commented on plugins/polio/js/src/components/LQAS-IM/ImSummary.tsx:
```diff
@@ -24,9 +30,9 @@ const style = theme => ({
-    warning: { color: WARNING_COLOR, textShadow: '1px 1px 2px black' },
+    pass: { color: OK_COLOR },
+    fail: { color: FAIL_COLOR },
+    warning: { color: 'rgb(255,196,53)' },
```
why not just modify the constant?

madewulf replied:
Because it's used at other places, where we were asked to keep yellow, and not orange 🤷‍♂️

olethanh commented:
feel like that will get complicated real quick :/


On Mon, Jan 17, 2022 at 2:36 PM Martin De Wulf ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In plugins/polio/js/src/components/LQAS-IM/ImSummary.tsx
> <https://github.com/BLSQ/iaso/pull/586#discussion_r786011548>:
>
> > @@ -24,9 +30,9 @@ const style = theme => ({
>
>      containerGrid: { justifyContent: 'space-evenly' },
>
>      centerText: { textAlign: 'center' },
>
>      boldText: { fontWeight: 'bold' },
>
> -    pass: { color: OK_COLOR, textShadow: '1px 1px 2px black' },
>
> -    fail: { color: FAIL_COLOR, textShadow: '1px 1px 2px black' },
>
> -    warning: { color: WARNING_COLOR, textShadow: '1px 1px 2px black' },
>
> +    pass: { color: OK_COLOR },
>
> +    fail: { color: FAIL_COLOR },
>
> +    warning: { color: 'rgb(255,196,53)' },
>
>
> Because it's used at other places, where we were asked to keep yellow, and
> not orange 🤷‍♂️
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BLSQ/iaso/pull/586#discussion_r786011548>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAUERHEBDRU6ZRH3YBZDSLUWQLNTANCNFSM5MEWRCYQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

olethanh committed:
IA-2150 fix timeout on derrived instance generation

by increasing the general timeout
The corresponding modification has been done on the load balancer for prod
This is a short term fix, the generation should be moved in a background task
